Dockerfile.test

```diff
@@ -13,15 +13,11 @@ ARG SCC_USERNAME
 ARG SCC_PASSWORD
 ARG SYSTEM_UUID
 
-RUN SUSEConnect -r $REGCODE
-RUN zypper --non-interactive ar http://download.opensuse.org/repositories/openSUSE:/Tools/SLE_15/openSUSE:Tools.repo
-RUN zypper --non-interactive --gpg-auto-import-keys ref
-RUN zypper --non-interactive up
-RUN zypper --non-interactive in -t pattern devel_osc_build
-RUN zypper --non-interactive install --no-recommend wget curl timezone \
-    gcc-c++ libffi-devel git-core zlib-devel libxml2-devel libxslt-devel libmariadb-devel \
-    mariadb-client mariadb ruby2.5-rubygem-bundler make build sudo ruby-devel nginx
-
+RUN mkdir /tmp/rmt-server
+WORKDIR /tmp/rmt-server
+ADD . /tmp/rmt-server
```

Comment:
Adding the whole directory probably would invalidate the cache if anything changes in it, which would also invalidate the cache for the following steps.

Reply:
any suggestions?

Reply:
Copy only `integration/install.sh` and not the whole directory.